bin/run-graphite-devel-server.py

```diff
@@ -8,6 +8,7 @@
 %prog [options] GRAPHITE_ROOT
 ''')
 option_parser.add_option('--port', default=8080, action='store', type=int, help='Port to listen on')
+option_parser.add_option('--interface', default="0.0.0.0", action='store', help='Interface to listen on')
```

Comment:
Man, that sticks out like a sore thumb now that you point it out. I normally go the other direction with everything in double quotes, so I know the importance of small details like this. Consistency in code is under-valued IMO.